src/Illuminate/Support/Stringable.php

```diff
@@ -234,18 +234,35 @@ public function endsWith($needles)
         return Str::endsWith($this->value, $needles);
     }
 
+    /**
+     * Alias for the `exactly` function.
+     *
+     * @param  \Illuminate\Support\Stringable|string  $value
+     * @param  bool  $ignoreCase
+     * @return bool
+     */
+    public function equals($value, $ignoreCase = false)
+    {
+        return $this->exactly($value, $ignoreCase);
+    }
+
     /**
      * Determine if the string is an exact match with the given value.
      *
      * @param  \Illuminate\Support\Stringable|string  $value
+     * @param  bool  $ignoreCase
      * @return bool
      */
-    public function exactly($value)
+    public function exactly($value, $ignoreCase = false)
     {
         if ($value instanceof Stringable) {
             $value = $value->toString();
         }
 
+        if ($ignoreCase) {
+            return strtolower($this->value) === strtolower($value);
```

Comment:
It would take `mb_` to support non-english case: `Ññ`, `Çç`, `Üü` etc.
```suggestion
            return mb_strtolower($this->value) === mb_strtolower($value);
```